packages/experimental/src/inbox-note/inbox-note.tsx

```diff
@@ -140,7 +140,10 @@ const InboxNoteCard: React.FC< InboxNoteProps > = ( {
 				renderToggle={ ( { onClose, onToggle } ) => (
 					<Button
 						isTertiary
-						onClick={ onToggle }
+						onClick={ ( event: React.MouseEvent ) => {
+							( event.target as HTMLElement ).focus();
```

Comment:
Safari doesn't focus on clicking according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#clicking_and_focus

Reply:
Thanks for explaining here!